config/checkstyle_checks.xml

```diff
@@ -35,6 +35,7 @@
     <property name="file" value="${checkstyle.suppressions.file}"/>
   </module>
   <module name="SuppressWarningsFilter"/>
+  <module name="SuppressWithPlainTextCommentFilter"/>
```

Comment:
This should be configured the same as `SuppressionCommentFilter`.

Reply:
Done